nlp-directives/src/test/java/io/cdap/google/directives/TestDirectives.java

```diff
@@ -0,0 +1,298 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.google.directives;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import io.cdap.wrangler.api.RecipePipeline;
+import io.cdap.wrangler.api.Row;
+import io.cdap.wrangler.test.TestingRig;
+import io.cdap.wrangler.test.api.TestRecipe;
+import io.cdap.wrangler.test.api.TestRows;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.AssumptionViolatedException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+/**
+ * Tests for the google nlp directives.
+ *
+ * To run the tests the path to a service account key must be provided.
+ * The service account key can be found on the Dashboard in the Cloud Platform Console.
+ * Make sure the account key has permission to access Natural Language API.
+ *
+ * mvn clean test -Dservice.account.file=<path-to-service-account-key-json>
+ *
+ */
+public class TestDirectives {
+  private static final Logger LOG = LoggerFactory.getLogger(TestDirectives.class);
+
+  private static final JsonParser PARSER = new JsonParser();
+  private static final String AUTHENTICATION_FILE = System.getProperty(
+    "service.account.file");
+
+  @BeforeClass
+  public static void initializeTests() {
+    try {
+      Assume.assumeNotNull(AUTHENTICATION_FILE);
+    } catch (AssumptionViolatedException e) {
+      LOG.warn("ETL tests are skipped. Please find the instructions on enabling it at " +
+                 "README.md");
```

Comment:
nit: this also looks like it can be on the previous line?

Reply:
Done